package.json

```diff
@@ -95,48 +95,48 @@
   "dependencies": {
     "chalk": "^2.4.1",
     "cross-spawn": "^6.0.5",
-    "enhanced-resolve": "^4.0.0",
-    "global-modules-path": "^2.1.0",
+    "enhanced-resolve": "^4.1.0",
+    "global-modules-path": "^2.3.0",
     "import-local": "^2.0.0",
     "interpret": "^1.1.0",
     "loader-utils": "^1.1.0",
-    "supports-color": "^5.4.0",
-    "v8-compile-cache": "^2.0.0",
-    "yargs": "^12.0.1"
+    "supports-color": "^5.5.0",
+    "v8-compile-cache": "^2.0.2",
+    "yargs": "^12.0.2"
   },
   "peerDependencies": {
     "webpack": "^4.x.x"
   },
   "devDependencies": {
-    "@commitlint/cli": "^7.0.0",
-    "@commitlint/config-angular": "^7.0.0",
-    "@commitlint/config-lerna-scopes": "^7.0.0",
-    "@commitlint/prompt-cli": "^7.0.0",
-    "@commitlint/travis-cli": "^7.0.0",
-    "@types/jest": "^23.1.1",
-    "@types/node": "^10.5.1",
+    "@commitlint/cli": "^7.1.2",
+    "@commitlint/config-angular": "^7.1.2",
+    "@commitlint/config-lerna-scopes": "^7.1.2",
+    "@commitlint/prompt-cli": "^7.1.2",
+    "@commitlint/travis-cli": "^7.1.2",
+    "@types/jest": "^23.3.2",
+    "@types/node": "^10.9.4",
     "bundlesize": "^0.17.0",
-    "codecov": "^3.0.2",
+    "codecov": "^3.1.0",
     "commitizen": "^2.10.1",
-    "conventional-changelog-cli": "^2.0.0",
+    "conventional-changelog-cli": "^2.0.5",
     "conventional-changelog-lint-config-cz": "^0.3.0",
     "cz-customizable": "^5.2.0",
-    "eslint": "^5.0.1",
-    "eslint-plugin-node": "^7.0.0",
+    "eslint": "^5.5.0",
+    "eslint-plugin-node": "^7.0.1",
     "husky": "^0.14.3",
-    "jest": "^22.4.4",
-    "jest-cli": "^23.0.1",
-    "lerna": "^3.0.0",
-    "lint-staged": "^7.1.2",
-    "nyc": "^12.0.1",
+    "jest": "^23.6.0",
+    "jest-cli": "^23.6.0",
+    "lerna": "^3.3.0",
+    "lint-staged": "^7.2.2",
+    "nyc": "^13.0.1",
     "prettier-eslint-cli": "^4.7.1",
     "rimraf": "^2.6.2",
     "schema-utils": "^1.0.0",
-    "tslint": "^5.10.0",
+    "tslint": "^5.11.0",
     "typedoc": "^0.12.0",
     "typedoc-plugin-monorepo": "^0.1.0",
     "typescript": "^2.9.2",
-    "webpack": "^4.8.3",
-    "webpack-dev-server": "^3.1.4"
+    "webpack": "^4.18.0",
```

Comment:
Shouldn't we remove these guys? Webpack and dev server. They should be dev peer dependencies. Well, that's outside of the scope the issue

Reply:
we need it to parse plugins. Needs to be sync with at least every major